src/main/scala/nl/knaw/dans/easy/fedoratobag/DDM.scala

```diff
@@ -62,8 +64,8 @@ object DDM extends DebugEnhancedLogging {
        { /* instructions for reuse not specified as such in EMD */ }
        { emd.getEmdCreator.getDcCreator.asScala.map(bs => <dc:creator>{ bs.getValue.trim }</dc:creator>) }
        { emd.getEmdCreator.getEasCreator.asScala.map(author => <dcx-dai:creatorDetails>{ toXml(author)} </dcx-dai:creatorDetails>) }
-       { dateCreated.map(node =>  <ddm:created>{ node.text }</ddm:created>) }
-       { dateAvailable.map(node =>  <ddm:available>{ node.text }</ddm:available>) }
+       { if (dateCreated.nonEmpty) dateCreated.toSeq.head.map(node =>  <ddm:created>{ parseDate(node.text) }</ddm:created>) }
+       { dateAvailable.map(node =>  <ddm:available>{ parseDate(node.text) }</ddm:available>) }
```

Comment:
Rather build `parseDate` into `getDateMap` and make `dateMap` a Map[String, Iterable[String]]): https://github.com/DANS-KNAW/easy-fedora-to-bag/blob/6c717c885b3420a1320c85e628821b5766afcc1d/src/main/scala/nl/knaw/dans/easy/fedoratobag/DDM.scala#L40-L46

Reply:
Didn't we decide to apply these correction in https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/
like ABR to ABR+?

Reply:
i want the bags produced by fedora-to-bag to be valid. These are not valid dates (it was botched up by the webui). The bags in the vault are already valid, so these problems don't exist there